platform/javascript/os_javascript.cpp

```diff
@@ -433,6 +555,9 @@ void OS_JavaScript::set_mouse_mode(OS::MouseMode p_mode) {
 	if (p_mode == MOUSE_MODE_VISIBLE) {
 
 		set_css_cursor(godot2dom_cursor(cursor_shape));
```

Comment:
No need for `set_css_cursor` since we're calling `set_cursor_shape` as well. Same in the `MOUSE_MODE_CAPTURED` case

Reply:
This is needed because if `Module.canvas.style.cursor` is equal `none` then `get_mouse_mode` will always return `MOUSE_MODE_HIDDEN` and the problem that you reported earlier will happens: https://github.com/godotengine/godot/pull/24040#pullrequestreview-188672636

Reply:
Comment would be good

Reply:
Comment added.